vision/objectdetection/postprocessor.go

```diff
@@ -51,6 +51,25 @@ func NewLabelFilter(labels map[string]interface{}) Postprocessor {
 	}
 }
 
+// NewLabelRenamer renames the labels in the input map from the key to the value.
+func NewLabelRenamer(labels map[string]string) Postprocessor {
+	return func(in []Detection) []Detection {
+		if len(labels) < 1 {
+			return in
+		}
+
+		for oldL, newL := range labels {
+			for i, d := range in {
```

Comment:
I dislike the fact that you have to iterate over the entire map every time, but I understand due to the prefix matching that you have to. I suggest an alternative though, because I think ALWAYS mapping to a prefix can cause problems (e.g. if my detector knows about "car" or "carrot" or "carnations" it will catch all of them even if I just wanted to replace "car")

The alternative:
- if there is a `*` at the end of the key in the label map, then you do a prefix match
- fi there is no `*` at the end of the key in the label map, you do an exact match

When first defining NewLabelRenamer, you split the map into two maps, the exact matcher map, and the prefix matcher map. Then you don't have to iterate over all of the  labels all the time.